static/js/app.js

```diff
@@ -281,11 +281,17 @@ _.templateSettings = {
           }
         })
 
-        // help
+        // about page
+        .state('about', {
+          url: '/about',
+          controller: 'AboutCtrl',
+          templateUrl: 'templates/partials/about.html'
+        })
+
         .state('help', {
-          url: '/help',
+          url: '/help/{page}',
           controller: 'HelpCtrl',
-          templateUrl: 'templates/partials/help.html'
+          templateUrl: 'templates/partials/help.html',
```

Comment:
Very minor... remove the trailing comma. In some browsers (though probably not ones we support) the trailing comma is illegal syntax.